src/ray/core_worker/core_worker.cc

```diff
@@ -2796,20 +2919,19 @@ CoreWorker::ListNamedActors(bool all_namespaces) {
     stream << "There was timeout in getting the list of named actors, "
               "probably because the GCS server is dead or under high load .";
     return std::make_pair(std::move(actors), Status::TimedOut(stream.str()));
-  } else if (!status.ok()) {
-    return std::make_pair(std::move(actors), status);
-  } else {
-    return std::make_pair(std::move(actors), status);
   }
+  if (!status.ok()) {
+    return std::make_pair(std::move(actors), std::move(status));
+  }
+  return std::make_pair(std::move(actors), std::move(status));
```

Comment:
these are both doing the same thing right?

Reply:
Good catch, updated.